docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  ms: &ms
+    build:
+      context: .
```

Comment:
Having the `docker-compose.yml` in the project root allows us to get metasploit up and running without having to do any configuration. As it stands you should be able to run `docker-compose run --rm --service-ports ms` on a fresh checkout, and be greeted with a `msfconsole` after the build completes.

If we move it out, users would have to set their `COMPOSE_FILE` environment variable. While not hard (`echo COMPOSE_FILE=./docker/docker-compose.yml >> .env`), its an extra step.

After testing, it looks like it changes the name of the docker-compose project, from: `metasploitframework_ms` to `docker_ms`. Not a big deal, we can fix that by specifying the `COMPOSE_PROJECT_NAME` environment variable in the the `.env` file.

What are your thoughts?